README.md

```diff
@@ -1,3 +1,58 @@
-# hello-microverse
+<!-- PROJECT DESCRIPTION -->
 
-This is the first basic project with microverse. The project does not intend to have professional codes, instead it should follow all the basic rules for project submission and follow no linter error along with github flow. 
+# 📖 Hello Microverse
+
+**Hello Microverse** is a project built to showcase basic tools understanding and integration for future projects. This is the first project with microverse with basic html & css. Included linters setup to check if the codes are neat and clean.
+
+## 🛠 Built With
+
+### Tech Stack
+
+<details open="">
+  <summary>Client</summary>
+  <ul dir="auto">
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features
+
+- **Shows Heading**
+- **Shows Paragraps**
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+
+To get a local copy up and running, follow these steps.
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+  cd your-folder-location
+
+  git clone git@github.com:sumon766/hello-microverse.git
+
+  OR
+
+  Download the zip file from https://github.com/sumon766/hello-microverse
+
+
+### Install
+
+No installation required. Just clone or download and open the index file in your browser
+
+<!-- AUTHORS -->
+
+## 👤 Author
+
+- GitHub: [@githubhandle](https://github.com/sumon766)
+- Twitter: [@twitterhandle](https://twitter.com/sumon766)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sumon766)
```

Comment:
- [ ] As the last code reviewer said kindly follow the instructions of the README template by Microverse. All main headers from the 👉🏽 [README template](https://github.com/microverseinc/readme-template) should be present in your repo for professional documentation.